app/components/Entity/components/ConfigureWorkflowInputs/components/Main/components/Stepper/components/Step/SequencingStep/components/ENASequencingData/components/CollectionSelector/utils.ts

```diff
@@ -35,36 +96,12 @@ export function preSelectColumnFilters(
 }
 
 /**
- * Builds validated column filters from preselected filter values.
- * Filters out columns that don't exist in the table and values that aren't available
- * in the column's faceted unique values.
- * @param table - Table instance.
- * @param preselectedFilters - Record of column IDs to arrays of desired filter values.
- * @returns Column filters state.
+ * Converts a value to an array of strings.
+ * @param value - The value to convert.
+ * @returns An array of strings.
  */
-function buildValidatedColumnFilters(
-  table: Table<ReadRun>,
-  preselectedFilters: Record<string, string[]>
-): ColumnFiltersState {
-  const columnFiltersState: ColumnFiltersState = [];
-
-  for (const [columnId, desiredValues] of Object.entries(preselectedFilters)) {
-    const column = table.getColumn(columnId);
-    if (!column) continue;
-
-    const availableValues = column.getFacetedUniqueValues();
-    const validValues: string[] = [];
-
-    // Only include values that exist in the column's faceted data.
-    for (const value of desiredValues) {
-      if (!availableValues.has(value)) continue;
-      validValues.push(value);
-    }
-
-    if (validValues.length === 0) continue;
-
-    columnFiltersState.push({ id: columnId, value: validValues });
-  }
-
-  return columnFiltersState;
+function toStringArray(value: unknown): string[] {
+  if (value == null) return [];
```

Comment:
Checking `==` is intentional here.

Reply:
LOL, no. I let Windsurf write that function 😱 .